model/schema/bulkload.yaml

```diff
@@ -135,7 +135,7 @@ slots:
     description: >-
       Indicates when the file was last loaded
     domain: BulkLoadFile
-    range: date  
+    range: datetime
```

Comment:
@sierra-moxon 
Is this intended to be a universal change (across the entire model) from a range of 'date' to 'datetime'? If so, there are two other locations still using 'date':
https://github.com/alliance-genome/agr_curation_schema/blob/main/model/schema/resource.yaml#L93
https://github.com/alliance-genome/agr_curation_schema/blob/main/model/schema/curationReport.yaml#L106
If those are fine to stay as 'date' and not 'datetime' then you can ignore.

Reply:
No definitely not, the new validation just does a better job of checking that the sample data we provide matches the schema.  So in our test data, we use datetime values (which is required by the java app layer, I think, right?), and our old validator wasn't checking that date was different from datetime.  Now it is so I fixed our schema to match the data.  

Other downstream apps (blue team work?) might require simple dates (we need tests to cover more of our schema), so this is a change that needs to happen on a case-by-case basis.